README.md

```diff
@@ -29,7 +29,36 @@ You can also install the bleeding edge version of `hssm` directly from this repo
 ```
 pip install git+https://github.com/lnccbrown/HSSM.git
 ```
+### Optional Installation
 
+**Dependency for graph() Function**
+
+Note: In addition to the installation of the main hssm class, there is an optional dependency for the graph() function. This dependency requires graphviz, which can be installed conveniently using conda with the following command:
+
+```
+conda install -c conda-forge python-graphviz
+```
+Alternatively, you have the option to install the graphviz binaries manually and then install the Python bindings using pip with the following command:
+
+```
+pip install graphviz
+```
+**Dependency for sampler="nuts_numpyro"**
+
+To utilize the nuts_numpyro sampler, please follow these steps:
+
+1. Install numpyro by executing the following command:
+```
+pip install numpyro
+```
+2. Import the necessary modules and configure the required settings:
+```
+import numpyro
```

Comment:
let's add something here also that points towards the jax documentation for installing gpu version of jax.